apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +137,71 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor name of the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "The days of the week for when the course meets",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description:
+        "The type of course level of the course (e.g. Lower Division, Upper Division, or Graduate)",
```

Comment:
Same comment as earlier. For future reference, it would be misleading to give an example ("Upper Division") which does not match the actual string returned by the API ("UpperDiv"). Since the variants are already specified, don't worry about doing it again.